src/components/panels/EventsPanel.vue

```diff
@@ -28,23 +28,25 @@
           {{ event.health || (100 - (event.currentScore / event.maximumScore) * 100).toFixed(2) }}% Remaining
         </b-badge>
 
-        <div v-if="event.rewards" class="text-center">Event Rewards:</div>
-        <div class="text-center d-inline" v-for="reward in event.rewards" :key="`rs-${reward.length}-${makeid()}`">
-          <b-badge v-for="item in reward.items" :key="`${item}-${makeid()}`" variant="success">
-            {{ item }}
-          </b-badge>
-          <b-badge v-for="item in reward.countedItems" :key="`${item}-${makeid()}`" variant="success">
-            {{ item }}
-          </b-badge>
-          <b-badge v-if="reward.credits" variant="info">{{ reward.credits }}{{ this.$t('currency.credAbbr') }}</b-badge>
-        </div>
-        <div class="text-center d-inline" v-for="step in event.interimSteps" :key="`rsi-${step.length}-${makeid()}`">
-          <b-badge v-for="item in step.reward.items" :key="`rsi-${item}-${makeid()}`" variant="success">
-            {{ item }}
-          </b-badge>
-          <b-badge v-for="item in step.reward.countedItems" :key="`rsi-${item}-${makeid()}`" variant="success">
-            {{ item }}
-          </b-badge>
+        <div class="text-center bottom-pad">
+          <div v-if="event.rewards">Event Rewards:</div>
+          <div class="text-center d-inline" v-for="reward in event.rewards" :key="`rs-${reward.length}-${makeid()}`">
+            <b-badge v-for="item in reward.items" :key="`${item}-${makeid()}`" variant="success">
+              {{ item }}
+            </b-badge>
+            <b-badge v-for="item in reward.countedItems" :key="`${item}-${makeid()}`" variant="success">
+              {{ item }}
+            </b-badge>
+            <b-badge v-if="reward.credits" variant="info">{{ reward.credits }}{{ this.$t('currency.credAbbr') }}</b-badge>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace `>{{·reward.credits·}}{{·this.$t('currency.credAbbr')·}}</b-badge` with `⏎··············>{{·reward.credits·}}{{·this.$t('currency.credAbbr')·}}</b-badge⏎············`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5261782)